runtime/vm/ModularityHashTables.c

```diff
@@ -66,18 +68,19 @@ moduleNameHashEqualFn(void *tableNode, void *queryNode, void *userData)
 	j9object_t queryNodeModuleName = queryNodeModule->moduleName;
 
 	Assert_VM_true(tableNodeModule->classLoader == queryNodeModule->classLoader);
-	
+
 	return javaVM->memoryManagerFunctions->j9gc_stringHashEqualFn(&tableNodeModuleName, &queryNodeModuleName, userData);
 }
 
-static UDATA 
-modulePointerHashFn(void *key, void *userData) 
+static UDATA
+modulePointerHashFn(void *key, void *userData)
 {
 	const J9Module** const modulePtr = (const J9Module**)key;
-	
+
 	return (UDATA)(*modulePtr);
```

Comment:
There a number of places like this with redundant parentheses. There's no other possible interpretation of this:
```c
	return (UDATA)*modulePtr;
```
See also lines 64, 67, 86, 87, 111 and 112.